drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_household_member.rb

```diff
@@ -14,5 +14,9 @@ class ReferralHouseholdMember < ::HmisExternalApis::HmisExternalApisBase
     enum(
       relationship_to_hoh: ::HudUtility.hud_list_map_as_enumerable(:relationship_to_ho_h_map)
     )
+
+    def self.where_is_h_oh
+      where(relationship_to_hoh: 'self_head_of_household')
+    end
```

Comment:
Shouldn't this be a scope rather than a class method? I'm not familiar with this pattern.